autopts/bot/common.py

```diff
@@ -152,21 +157,18 @@ def run_test_cases(self):
             _args[config].excluded = []
             _args[config].test_cases = []
 
-            remaining_test_cases = []
```

Comment:
Please take a note that in python this line: `remaining_test_cases = filtered_test_cases` will assign a reference to a variable, not create a copy of the list. Because of that, `remaining_test_cases.remove(tc)` will result in skipping next test case from the list. Here is an example you can run to see what will happen:
```python
test_cases = [
    'GAP/BROB/BCST/BV-01-C',
    'GAP/BROB/BCST/BV-02-C',
    'GAP/BROB/BCST/BV-03-C',
    'GAP/BROB/BCST/BV-04-C',
    'GAP/BROB/BCST/BV-05-C',
    'GAP/BROB/BCST/BV-06-C',
    'GAP/BROB/BCST/BV-07-C',
]

test_cases_not_deep_copy = test_cases

for tc in test_cases:
    print(tc)
    if tc == 'GAP/BROB/BCST/BV-04-C':
        test_cases.remove(tc)

print('test_cases_not_deep_copy:', test_cases_not_deep_copy)
print('test_cases:', test_cases)
```
In the example the 'GAP/BROB/BCST/BV-05-C' is skipped in `for` enumeration, because of the remove(), and the `test_cases_not_deep_copy` does not have 'GAP/BROB/BCST/BV-04-C' any more too.